apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js

```diff
@@ -14,6 +13,10 @@ import { getSectionName } from 'calypso/state/ui/selectors';
 import { whatsNewQueryClient } from '../../common/what-new-query-client';
 import CalypsoStateProvider from './CalypsoStateProvider';
 
+function PinnedItems( { scope, ...props } ) {
```

Comment:
This deservers a comment explaining why we avoid using interface. Because it's a bit of a hack.

I'd say: Implement `PinnedItems` to avoid importing `@wordpress/interface`. Because `@wordpress/interface` depends on `@wordpress/preferences` which is not always available outside the editor, causing the script to not be enqueued due to the missing dependency.

Reply:
✅